app/schemas/user.py

```
@@ -2,13 +2,34 @@
 from pydantic import BaseModel, ConfigDict, EmailStr, Field, field_validator
 
 from app.core.security import utc_now
 from app.models.enums import SubscriptionType
 
 
+def validate_phone_region(v: str) -> str:
+    if not isinstance(v, str):
+        return v
+
+    if not v.startswith("+"):
+        raise ValueError("Phone region must start with '+'.")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)